packages/ember-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -27,6 +27,20 @@ export default function TransformAttrsToProps() {
   this.syntax = null;
 }
 
+function isAttrs(node) {
+  if (node.parts[0] === 'attrs') {
+    return true;
+  }
+
+  let [_this, attrs] = node.parts;
+
+  if (_this === null && attrs === 'attrs') {
+    node.parts.shift();
+    node.original = node.original.slice(5);
+    return true;
+  }
```

Comment:
should we have the ability to warn when we do this so that people can fix their code bases?